app/views/companies/show.html.haml

```diff
@@ -37,6 +37,9 @@
     %b #{t('.region')}:
     = @company.region.name if @company.region
 
+  - if current_user.try(:admin)
```

Comment:
I suspect that's because you're using an attribute reference (```admin```) rather than a method (```admin?```).  No change needed here - just a matter of style and preference, I guess.